packages/esm-patient-labs-app/src/lab-orders/add-lab-order/lab-order.ts

```diff
@@ -0,0 +1,19 @@
+import { LabOrderBasketItem } from '../api';
+import { TestType } from './useTestTypes';
+
+// See the Urgency enum in https://github.com/openmrs/openmrs-core/blob/492dcd35b85d48730bd19da48f6db146cc882c22/api/src/main/java/org/openmrs/Order.java
+export const priorityOptions = [
+  { value: 'ROUTINE', label: 'Routine' },
+  { value: 'STAT', label: 'Stat' },
+];
+// TODO add priority option `{ value: "ON_SCHEDULED_DATE", label: "On scheduled date" }` once the form supports a date.
```

Comment:
`ON_SCHEDULED_DATE` is probably more relevant for medications. Lab orders usually aren't quite so tightly timed.